src/api/init.ts

```diff
@@ -327,18 +319,6 @@ export async function startApiServer(opts: {
     next(error);
   }) as express.ErrorRequestHandler);
 
-  app.use(
```

Comment:
Have you tested that error responses that get set in the `express.ErrorRequestHandler` handler above this are logged correctly? IIUC the pino logging middleware is configured before the error handling middleware so I'm not sure that it will be aware.